### PR TITLE
Update newsletter sponsorship pricing

### PR DESCRIPTION
**Pull request by ericholscher**
Also does some content updates and minor refactoring.

----
📚 Documentation preview 📚: https://writethedocs-www--2162.org.readthedocs.build/

### CODE REVIEW

File: docs/include/conf/sponsorship/demographics.rst [hunk 1]
```diff
@@ -11,8 +11,8 @@ For those people, they have a range of job titles.
 The most common job title is **Technical Writer**,
 but we have many other people who attend our conferences:
```
Comment (plaindocs) [inline docs/include/conf/sponsorship/demographics.rst]:
```suggestion
The most common job title is **Technical Writer**,
but people with many other roles also attend our conferences:

```

Driveby not in the PR, soz. 
Also, shouldn't we delete this and use `docs/include/demographics.rst` ?

File: docs/sponsorship/newsletter.rst [hunk 2]
```diff
@@ -16,47 +17,38 @@ Audience
...
 
-* Over 5,000 subscribers
-* 37% open rate
-* 6% click rate
-* 70% of subscribers are based in the United States
-
-We don't have specific demographic data for our newsletter,
-but our conferences are made up of:
+* Over {{ newsletter_subs }} subscribers
+* 42% open rate
```
Comment (plaindocs) [inline docs/sponsorship/newsletter.rst:21]: Is it worth highlight just how high this number is compared to industry average? @goatlady

File: docs/sponsorship/newsletter.rst [hunk 3]
```diff
@@ -16,47 +17,38 @@ Audience
...
 
-- Technical Writers (60%)
-- Developers (10%)
-- Support Staff (10%)
-- Managers (10%)
-- Community Contributors, Enthusiasts & Other Folks (10%)
+The best idea of who is visiting our site is the information we have from our conferences.
```
Comment (plaindocs) [inline docs/sponsorship/newsletter.rst]:
```suggestion
We don't exact demographic data for our newsletter subscribers, but according to the data gathered from our conferences, our audience is made up of:
```

File: docs/sponsorship/newsletter.rst [hunk 4]
```diff
@@ -16,47 +17,38 @@ Audience
...
+Our audience is made up of:
```
Comment (plaindocs) [inline docs/sponsorship/newsletter.rst]:
```suggestion

```

File: docs/sponsorship/newsletter.rst [hunk 5]
```diff
@@ -16,47 +17,38 @@ Audience
...
 
+.. include:: /include/demographics.rst
 
 Sponsored entry format
 ----------------------
 
-The current format is:
+Our sponsorship format is similar to our normal newsletter entries.
```
Comment (plaindocs) [inline docs/sponsorship/newsletter.rst]:
```suggestion
Our sponsorship format is similar to our normal newsletter entries, with an added company logo.
```

**Review by plaindocs — APPROVED** [review]
Nice updates, I've suggested a few improvements.

File: docs/sponsorship/newsletter.rst (hunk 2 above)
Comment (goatlady) [reply docs/sponsorship/newsletter.rst:21]: The problem is that open rates are kind of dead as a metric since Apple started blocking tracking pixels by default and Google started caching images from bulk senders a couple of years ago. Your open rate is almost certainly much higher than that, but because most of the content is in the email itself rather than outbound links, MailChimp can't count the click-throughs as confirmed opens. 

Having said that, regardless of industry, an open rate of over 30% is considered excellent. There's an industry breakdown from 2022 data here: https://www.campaignmonitor.com/resources/knowledge-base/what-are-good-email-metrics/
